LongWarOfTheChosen/Config/XComWeaponAbilities.ini

```diff
@@ -0,0 +1,12 @@
+;A configuration File

```

Comment:
Why are these not in `XComGameData_WeaponData.ini`? That's where the SMG mobility bonus stuff is. I think it you make sense to keep them in the same config file.

Reply:
Ok I will move the bullpup config information to the XComGameData_WeaponData.ini file for consistency.

Reply:
Configs for this are now in GameData_WeaponData.